### ISSUE TITLE
Missing #include in util.cpp

### ISSUE THREAD

**Issue by anderstn** [sass/libsass#1144]
In commit 7e1b94b94eb60833dcd96f572f1357b924b62183 a macro using stderr was added in util.cpp however stdio.h was not included. As a result the file won't compile on Windows.

Broken code:
```c++
#include<stdint.h>
#include "ast.hpp"
#include "util.hpp"
#include "prelexer.hpp"
#include "utf8/checked.h"

namespace Sass {

  #define out_of_memory() do {                    \
      fprintf(stderr, "Out of memory.\n");    \
      exit(EXIT_FAILURE);                     \
    } while (0)

  /* Sadly, sass_strdup is not portable. */
  char *sass_strdup(const char *str)
  {
    char *ret = (char*) malloc(strlen(str) + 1);
    if (ret == NULL)
      out_of_memory();
    strcpy(ret, str);
    return ret;
  }
  
  ...
}
```

Code that compiles:
```c++
#include<stdint.h>
#include<stdio.h>
#include "ast.hpp"
#include "util.hpp"
#include "prelexer.hpp"
#include "utf8/checked.h"

namespace Sass {

  #define out_of_memory() do {                    \
      fprintf(stderr, "Out of memory.\n");    \
      exit(EXIT_FAILURE);                     \
    } while (0)

  /* Sadly, sass_strdup is not portable. */
  char *sass_strdup(const char *str)
  {
    char *ret = (char*) malloc(strlen(str) + 1);
    if (ret == NULL)
      out_of_memory();
    strcpy(ret, str);
    return ret;
  }
  
  ...
}
```

**Comment by xzyfer** [comment]
This was also reported in https://github.com/sass/node-sass/issues/892 prevent compilations on FreeBSD

**Comment by xzyfer** [comment]
Fixed in [3.2.1](https://github.com/sass/libsass/releases/tag/3.2.1)
